### DISCUSSION TITLE
A Gamified Green Software for Practitioners Event?

### DISCUSSION THREAD

**Discussion by greenhsu123** [Green-Software-Foundation/discussions#19] [Community Working Group]
I propose a week-long event to drive the take-up of the Green Software for Practitioners course. During the week, we use social, meetups (study groups?), and virtual teaching of the principles of green software to encourage people to complete the course. 

I think a date that has [environmental significance](https://en.wikipedia.org/wiki/List_of_environmental_dates) that also falls in the summer would be ideal. 

Thoughts?

We should also find out when each member organisation have its summer interns to maximise awareness among the student population. 

cc: @jawache @tmcclell  @aecurrie @liyamath

**Comment by LiyaMath** [comment]
@greenhsu123 That's a good idea. how about interns+university students? And I think clubbing this with this discussion on Engaging with higher educational institutes in the UK could give us a large audience for the Green Software for Practitioners course.

**Comment by jawache** [comment]
I would suggest with the time remaining to just focus on one thing, be it a social media thing, a virtual webinar or something else - I think if it was one thing there is enough time to put peoples attention to it.

cc/ @NAMRATA-WOKE for her thoughts, It's super tight in terms of timeline, if something is decided here will probably need to be done and ready in 1-2 weeks to give enough time to get the word out.

**Comment by greenhsu123** [comment]
ah, sorry about the confusion, the above proposal is _NOT_ for Earth Week 2023, but for a later date in the Sumner 2023.

**Reply by jawache** [reply]
Ahhh wow, my brain read Earth Week somewhere in your text but it's not there at all!

**Reply by jawache** [reply]
So in response, phew, yes that works better!

**Comment by NAMRATA-WOKE** [comment]
@jawache, would you like me to mention this in my comms update email to see what information they can share with us re: summer internship programs at their company?

**Reply by jawache** [reply]
Sure that sounds like a good first step, as long as we don't set any expectations, we're still exploring here :) If there is any interest I'd get everyone connected with @greenhsu123 and @LiyaMath as a first step.

**Comment by tmcclell** [comment]
Discussion around prizes with sustainability in mind or guest appearance for the podcast, digital giveaways. Approved by the Community WG. @adamj89  - Add to timelines please (summer interns)
